pymc3/ode/utils.py

```diff
@@ -27,26 +29,27 @@ def augment_system(ode_func, n, m):
     """
 
     # Present state of the system
-    t_y = tt.vector("y", dtype=theano.config.floatX)
-    t_y.tag.test_value = np.zeros((n,))
+    t_y = tt.vector("y", dtype='float64')
+    t_y.tag.test_value = np.zeros((n,), dtype='float64')
     # Parameter(s).  Should be vector to allow for generaliztion to multiparameter
-    # systems of ODEs.  Is m dimensional because it includes all ode parameters as well as initical conditions
-    t_p = tt.vector("p", dtype=theano.config.floatX)
-    t_p.tag.test_value = np.zeros((m,))
+    # systems of ODEs.  Is m dimensional because it includes all initial conditions as well as ode parameters
+    t_p = tt.vector("p", dtype='float64')
+    t_p.tag.test_value = np.zeros((m,), dtype='float64')
     # Time.  Allow for non-automonous systems of ODEs to be analyzed
-    t_t = tt.scalar("t", dtype=theano.config.floatX)
+    t_t = tt.scalar("t", dtype='float64')
     t_t.tag.test_value = 2.459

```

Comment:
does 2.459 have a story??

Reply:
I have no idea. But there are also some 0es in the test values that can cause `nan`s during test value computation. But this depends on the ODEs, so I guess there's no single best way to choose them.